File: CRUD+Métier/CRUD/src/crud/pidev/entities/Stade.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package crud.pidev.entities;

/**
 *
 * @author esprit
 */
public class Stade {

    private int id_stade;
    private String nom_stade;
    private String ville;
    private String adresse;
    private int capacite;
    private int dispo;

    public Stade() {

    }

    public Stade( String n, String v, String a, int c, int d) {
        //this.id_stade = id;
        this.nom_stade = n;
        this.ville = v;
        this.adresse = a;
        this.capacite = c;
        this.dispo = d;

    }

    /**
     * ********************* Les getteurs ********************
     */
    public int getId_stade() {
        return id_stade;
    }

    public String getNom_stade() {
        return nom_stade;
    }

    public String getVille() {
        return ville;
    }

    public String getAdresse() {
        return adresse;
    }

    public int getCapacite() {
        return capacite;
    }

    public int getDispo() {
        return dispo;
    }

    /**
     * *********************** les setteurs ***************
     */
    public void setId_stade(int id_stade) {
        this.id_stade = id_stade;
    }

    public void setNom_stade(String nom_stade) {
        this.nom_stade = nom_stade;
    }

    public void setVille(String ville) {
        this.ville = ville;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public void setCapacite(int capacite) {
        this.capacite = capacite;
    }

    public void setDispo(int dispo) {
        this.dispo = dispo;
    }

    public String toString() {

        return ""
                + "ID Stade" + this.id_stade
                + "Nom Stade" + this.nom_stade
                + "Ville" + this.ville
                + "Adresse" + this.adresse
                + "Capacite" + this.capacite
                + "Disponibilite" + this.dispo;
        
    }

}

File: src/Entity/Stade.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Stade
 *
 * @ORM\Table(name="stade")
 * @ORM\Entity
 */
class Stade
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_stade", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idStade;

    /**
     * @var int
     *
     * @ORM\Column(name="id_org", type="integer", nullable=false)
     */
    private $idOrg;

    /**
     * @var string
     *
     * @ORM\Column(name="nom_stade", type="string", length=100, nullable=false)
     */
    private $nomStade;

    /**
     * @var string
     *
     * @ORM\Column(name="ville", type="string", length=100, nullable=false)
     */
    private $ville;

    /**
     * @var string
     *
     * @ORM\Column(name="adresse", type="string", length=100, nullable=false)
     */
    private $adresse;

    /**
     * @var int
     *
     * @ORM\Column(name="capacite", type="integer", nullable=false)
     */
    private $capacite;

    /**
     * @return int
     */
    public function getIdStade(): int
    {
        return $this->idStade;
    }

    /**
     * @param int $idStade
     */
    public function setIdStade(int $idStade): void
    {
        $this->idStade = $idStade;
    }

    /**
     * @return int
     */
    public function getIdOrg(): int
    {
        return $this->idOrg;
    }

    /**
     * @param int $idOrg
     */
    public function setIdOrg(int $idOrg): void
    {
        $this->idOrg = $idOrg;
    }

    /**
     * @return string
     */
    public function getNomStade(): string
    {
        return $this->nomStade;
    }

    /**
     * @param string $nomStade
     */
    public function setNomStade(string $nomStade): void
    {
        $this->nomStade = $nomStade;
    }

    /**
     * @return string
     */
    public function getVille(): string
    {
        return $this->ville;
    }

    /**
     * @param string $ville
     */
    public function setVille(string $ville): void
    {
        $this->ville = $ville;
    }

    /**
     * @return string
     */
    public function getAdresse(): string
    {
        return $this->adresse;
    }

    /**
     * @param string $adresse
     */
    public function setAdresse(string $adresse): void
    {
        $this->adresse = $adresse;
    }

    /**
     * @return int
     */
    public function getCapacite(): int
    {
        return $this->capacite;
    }

    /**
     * @param int $capacite
     */
    public function setCapacite(int $capacite): void
    {
        $this->capacite = $capacite;
    }


}
